docs/UserGuide.md

```diff
@@ -1,46 +1,57 @@
-| [FAQ](#faq)                         | If you have any questions about the application                         |
-| [Known Issues](#known-issues)       | If you have a technical problem. We may have the solution!              |
-| [Command Summary](#command-summary) | If you would like to find more information about a specific command     |
+# CareerSync User Guide
+Welcome to <span style="color: #f66a0a;">CareerSync</span>'s  User Guide! <span style="color: #f66a0a;">CareerSync</span> 
+is an **internship application manager designed to simplify the management of internship applications**. It is designed to help you **keep track of all your
+internship applications in one place**, so that you can focus on preparing for your interviews and securing your dream internship.
+
+CareerSync features a sleek and user-friendly interface primarily driven by command inputs through **Command Line Interface (CLI)**. 
+For those who are **proficient typists**, CareerSync facilitates quicker completion of internship applications 
+compared to conventional GUI-based applications. If you are a NUS Computer Science student who is looking to apply for
+many internships, CareerSync is practically made just for you!
+
+In this user guide, we will guide you through all the functionalities of CareerSync, so that you can make the most out of this app. 
+From a Year 1 rookie who just started applying for internships, to a Year 4 veteran who has been through multiple
+internship applications, our guide will ensure that you can use CareerSync to its full potential.
+
+### What do I need to know for this User Guide?
+This user guide is designed to be approachable for beginners, requiring no previous familiarity with administrative tools
+or command interfaces, and is self-sufficient. Our [Quick Start](#quick-start) section will guide you through the installation
+of all the necessary tools, even if you have never used a CLI before.
+
+### How to use this User Guide?
+This user guide is split into these primary sections: [Quick Start](#quick-start), [Interface Overview](#interface-overview), [Tutorial](#tutorial), 
+[Commands](#commands), [FAQ](#faq), [Known Issues](#known-issues),[Command Summary](#command-summary), [Fields Summary](#fields-summary) and [Glossary](#glossary). 
+We recommend that you navigate our User Guide in this order:
+1. Go through the [Quick Start](#quick-start) section to get started with the installation of CareerSync.
+2. Take a look at [Interface Overview](#interface-overview) to understand the elements of our GUI.
+3. Follow the instructions in [Tutorial](#tutorial) to understand the basic commands and how to use them.
+4. Read through the [Commands](#commands) section to understand all the commands and how to use them.
+5. Read through the [Fields Summary](#fields-summary) section to understand all the fields and their restrictions.
```

Comment:
Would this make the **table of contents - when should I use this** part redundant?

Reply:
that is removed